packages/web/components/Setup/SetupTimeZone.tsx

```diff
@@ -46,6 +48,14 @@ export const SetupTimeZone: React.FC<SetupTimezoneProps> = ({
     onNextPress();
   };
 
+  const [isComponentMounted, setIsComponentMounted] = useState(false);
+
+  useEffect(() => setIsComponentMounted(true), []);
+
+  if (!isComponentMounted) {
```

Comment:
Why do we need to do this ?

Reply:
A server side rendered page was trying to access `<SelectTimeZone>` component, which is a part of the design-system, and was not found until the client loaded. The builds and deploys failed and when I introduced this change they passed. I don't know how else to solve this problem, do you perhaps have any suggestion?

Reply:
Interesting. I tried a few different things and nothing fixed that issue. :/

Reply:
Yep, it's quite peculiar 🤷